### PR TITLE
fix: oneflow demo and docker test

### LINKED ISSUE #3151: deploy failed when last join
# Error
https://github.com/4paradigm/OpenMLDB/actions/runs/4341576688/jobs/7581344528#step:5:1253

`Error: [2000] new ttl type kLatestTime doesn't match the old ttl type kAbsoluteTime in table action`
about table action:
```
last join `action` as `action_reqId` on `flattenRequest`.`reqId` = `action_reqId`.`reqId`
```

`Error: [2000] new ttl type kAbsAndLat doesn't match the old ttl type kLatestTime in table flattenRequest`
about table flattenRequest:
```
flattenRequest_user_id_eventTime_0s_14d_200 as (partition by `user_id` order by `eventTime` rows_range between 14d preceding and 0s preceding MAXSIZE 200)
```

# Desc

Deploy sql will failed when it has `last join`. We can reproduce it by oneflow demo case in 0.7.2 docker.
```
docker run --name=openmldb -it ...
cat <<EOF | docker exec --interactive openmldb bash
/work/init.sh
/work/openmldb/bin/openmldb --zk_cluster=127.0.0.1:2181 --zk_root_path=/openmldb --role=sql_client < /work/oneflow_demo/sql_scripts/create_tables.sql
/work/openmldb/bin/openmldb --zk_cluster=127.0.0.1:2181 --zk_root_path=/openmldb --role=sql_client < /work/oneflow_demo/sql_scripts/deploy.sql
EOF
```

# TODO
1. make deploy succeed
2. get the error if deploy failed in compose test
3. oneflow doc: fix the oneflow pip version and more info about cudnn

### PR DESCRIPTION
**Pull request by vagetablechicken**
close #3151

### CODE REVIEW

File: docs/zh/use_case/JD_recommendation.md [hunk 1]
```diff
@@ -34,19 +34,21 @@ OneFlow工具依赖GPU的强大算力，所以请确保部署机器具备Nvidia
...
 pip install psutil petastorm pandas sklearn xxhash "tritonclient[all]" geventhttpclient tornado
 ```
+```{note}
+我们仅验证了oneflow 0.8.0版本，如果你使用其他版本，可能会遇到问题。下载版本中`cu`后缀对应的CUDA版本，例如`cu112`对应CUDA 11.2。
+```TODO: conda env oneflow-0426, test on it
 
 拉取Oneflow-serving镜像：
 ```bash
 docker pull oneflowinc/oneflow-serving:nightly
+docker pull oneflowinc/oneflow-serving:v0.1.0 is OK?
```
Comment (vagetablechicken) [inline docs/zh/use_case/JD_recommendation.md]: check 0.1.0

File: docs/zh/use_case/JD_recommendation.md [hunk 2]
```diff
@@ -34,19 +34,21 @@ OneFlow工具依赖GPU的强大算力，所以请确保部署机器具备Nvidia
 ```bash
 conda create -y -n oneflow python=3.9.2
 conda activate oneflow
-pip install -f https://staging.oneflow.info/branch/master/cu112 --pre oneflow
+pip install -f https://release.oneflow.info oneflow==0.8.0+cu112
 pip install psutil petastorm pandas sklearn xxhash "tritonclient[all]" geventhttpclient tornado
 ```
+```{note}
+我们仅验证了oneflow 0.8.0版本，如果你使用其他版本，可能会遇到问题。下载版本中`cu`后缀对应的CUDA版本，例如`cu112`对应CUDA 11.2。
+```TODO: conda env oneflow-0426, test on it
```
Comment (vagetablechicken) [inline docs/zh/use_case/JD_recommendation.md]: fix it

File: docs/zh/use_case/JD_recommendation.md (hunk 1 above)
Comment (vagetablechicken) [reply docs/zh/use_case/JD_recommendation.md]: only nightly works

File: docs/zh/use_case/JD_recommendation.md (hunk 2 above)
Comment (vagetablechicken) [reply docs/zh/use_case/JD_recommendation.md]: use 0.9.0

**Review by dl239 — APPROVED** [review]

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**